internal/db/queries.sql

```diff
@@ -33,8 +33,8 @@ WITH dequeued AS (
 	UPDATE mergestat.repo_sync_queue SET status = 'RUNNING'
 	WHERE id IN (
 		SELECT id FROM mergestat.repo_sync_queue
-		WHERE status = 'QUEUED'
-		ORDER BY repo_sync_queue.created_at ASC, repo_sync_queue.id ASC LIMIT 1 FOR UPDATE SKIP LOCKED
+		WHERE status = 'QUEUED' AND priority >=0
+		ORDER BY repo_sync_queue.priority DESC,repo_sync_queue.created_at ASC, repo_sync_queue.id ASC LIMIT 1 FOR UPDATE SKIP LOCKED
```

Comment:
```suggestion
		ORDER BY repo_sync_queue.priority DESC, repo_sync_queue.created_at ASC, repo_sync_queue.id ASC LIMIT 1 FOR UPDATE SKIP LOCKED
```

just adding a space here